server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -72,22 +69,24 @@
   private static final int NAME_MAX_LENGTH = 200;
 
   private final NewUserNotifier newUserNotifier;
-  private final Settings settings;
   private final DbClient dbClient;
   private final UserIndexer userIndexer;
   private final System2 system2;
+  private final OrganizationFlags organizationFlags;
   private final DefaultOrganizationProvider defaultOrganizationProvider;
   private final OrganizationCreation organizationCreation;
+  private final DefaultGroupFinder defaultGroupFinder;
 
-  public UserUpdater(NewUserNotifier newUserNotifier, Settings settings, DbClient dbClient, UserIndexer userIndexer, System2 system2,
-    DefaultOrganizationProvider defaultOrganizationProvider, OrganizationCreation organizationCreation) {
+  public UserUpdater(NewUserNotifier newUserNotifier, DbClient dbClient, UserIndexer userIndexer, System2 system2, OrganizationFlags organizationFlags,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)